scipy/signal/_czt.py

```diff
@@ -0,0 +1,549 @@
+# This program is public domain
+# Authors: Paul Kienzle, Nadav Horesh
+"""
+Chirp z-transform.
+
+We provide two interfaces to the chirp z-transform: an object interface
+which precalculates part of the transform and can be applied efficiently
+to many different data sets, and a functional interface which is applied
+only to the given data set.
+
+Transforms
+----------
+
+CZT : callable (x, axis=-1) -> array
+   Define a chirp z-transform that can be applied to different signals.
+ZoomFFT : callable (x, axis=-1) -> array
+   Define a Fourier transform on a range of frequencies.
+
+Functions
+---------
+
+czt : array
```

Comment:
This is a slightly odd type annotation.  Do we use it this way elsewhere?  It is not correct that `czt` is an array, so I'd write: `czt : callable(...) -> array`. Not sure what the `CZT` definition above should be; it's a bit confusing right now.

Reply:
I don't see this type of docstring in any other module.  Do you have any examples of what it should look like?